src/app/CMakeLists.txt

```diff
@@ -249,6 +214,16 @@ add_executable(MuseScoreStudio
     internal/guiapp.h
 )
 
+if (OS_IS_WASM)
+    #!! NOTE Fake module with a qml file that lists all Qml/Quick modules,
+    #!! needed for correct deployment for Wasm.
+    qt_add_qml_module(MuseScoreStudio
```

Comment:
See also the approach we used for iOS: https://github.com/cbjeukendrup/MuseScore/commit/29e01e7b731ec275bc9eb9784ed3cd5c6e0c708e (assuming Qt for WebAssembly is also statically linked, like on iOS)

Reply:
applied your solution